### ISSUE TITLE
Modify Exercise

### ISSUE THREAD

**Issue by JMarco27** [thexerteproject/xerteonlinetoolkits#1101]
Could the following changes be considered?

1. Could the **Initial Text** property be an Optional Property?  Sometimes it's useful to see the original text after the learner has started editing it in the **Answer Prompt** box, but this may be unnecessary, and the learner may be confused if presented with two identical boxes.

2. Is it possible for the **Model Answer** property to be a formattable text box for the author?  If you set a task requiring the learner to identify and correct some grammatical/spelling/factual errors in the text, it's useful, in the Model Answer, to be able to underline, put in bold or highlight the parts of the text which need to be changed. **Model Answer** here is essentially feedback, so to be consistent with other Interactivity page types, perhaps it should be re-labelled **Feedback**, and include all the usual editor features?

**Comment by JMarco27** [comment]
Could someone assess these requests? Are they useful enhancements?  How feasible are they?

**Comment by FayCross** [comment]
@JMarco27 all good ideas - I've just done the following for v3.12:
- model answer field is now WYSIWYG
- new 'Display as plain text' property is checked by default so unless unchecked the model answer field will remain plain text to match answer box
- new 'Feedback' optional property
- if 'Initial text' is left empty then the initial text box will not show

**Comment by JMarco27** [comment]
Hi Fay

Thanks for making the changes to the Modify Exercise.  This is really
useful.  Interestingly, on my personal installation I tend to use the Flat
Blue theme (because it is similar to the theme Takeshi adesigned for
Reading), but it actually displays better in Modern Django.

https://xotbeta.dlearning.nl/play.php?template_id=140

I notice the Responsive Text tick box, in Project Settings, has gone. I
can't see anything on this in the Release Notes.  I tended to untick
Responsive Text because it created problems with font size, particularly in
Gap Fill tasks, where text between gaps displayed in different sizes, and
the problem was exacerbated when I used Ctrl/- to zoom out. Have these
display issues been resolved?

Many thanks. I really appreciate your impressively swift response. Is there
anything specific you would like me to test for 3.12?

Best wishes

Jonathan
